packages/react-table/src/components/Table/TableTypes.tsx

```diff
@@ -1,10 +1,10 @@
-import { DropdownItemProps } from '@patternfly/react-core';
+import { ButtonProps } from '@patternfly/react-core/dist/esm/components/Button';
 import { formatterValueType, ColumnType, RowType, RowKeyType, HeaderType } from './base';
 import { SortByDirection } from './SortColumn';
-import { DropdownDirection, DropdownPosition } from '@patternfly/react-core/dist/esm/deprecated/components';
+import { DropdownItemProps } from '@patternfly/react-core/dist/esm/components/Dropdown';
+import { DropdownDirection, DropdownPosition } from '@patternfly/react-core/dist/esm/deprecated/components/Dropdown';
```

Comment:
We may want to make a note to clean up the remaining usages of the old dropdown. Looks like it's still referenced in `IColumn` here and `TdActionsType` in `base/types`.

Reply:
@thatblindgeye is this part of your PR for updating demos to use the new dropdown?

Reply:
Yeah that's something that I should be able to incorporate into #8839